### PR TITLE
Release v0.9.0

### PR DESCRIPTION
**Pull request by sjcobb**
- bump package versions to 0.9.0, update changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,30 @@
 # Changelog
 
+# 0.9.0 / 2022-10-04
+
+- [FEATURE] New ListVariable plugin for StaticListVariables #547
+- [FEATURE] New Prometheus plugin for ListVariable (LabelName, LabelValues) #565
+- [FEATURE] Variable chaining support for Prometheus variables #579
+- [FEATURE] New editable Markdown panel #553, #555, #563
+- [FEATURE] New datasource spec implemented #554, #577
+- [ENHANCEMENT] Monorepo / build tooling improvements #550, #567, #578, #585, #586
```
Comment (Nexucis) [inline CHANGELOG.md:10]: I would also add here #583.
#578 seems a bit odd comparing to the others PRs because others are really focus on improving the CI.

**Review by Nexucis — APPROVED** [review]
Overall looks good to me. Let's home the release will be ok ! 🤞

**Comment by Nexucis** [comment]
Ah wait I would like to merge #589 and have it in this version. That will complete the rework around the CI

File: CHANGELOG.md (hunk 1 above)
Comment (LukeTillman) [reply CHANGELOG.md]: I fixed it!

**Comment by Nexucis** [comment]
if someone can look at #589 and maybe approve it before releasing the code, that would cool :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,18 +1,39 @@
...
+# 0.9.0 / 2022-10-04
+
+- [FEATURE] New ListVariable plugin for StaticListVariables #547
+- [FEATURE] New Prometheus plugin for ListVariable (LabelName, LabelValues) #565
+- [FEATURE] Variable chaining support for Prometheus variables #579
+- [FEATURE] New editable Markdown panel #553, #555, #563
+- [FEATURE] New datasource spec implemented #554, #577
+- [ENHANCEMENT] Upgrade react-query to v4 #578
+- [ENHANCEMENT] Monorepo / build tooling improvements #550, #567, #583, #585, #586
+- [ENHANCEMENT] Update Perses Header, refactor embeddable components #580, 583
```
Comment (Nexucis) [inline CHANGELOG.md:12]: oh there is a glitch here. I guess 583 should be removed

**Review by LukeTillman — APPROVED** [review]
:shipit:

**Pull request merged**